antlr/C.g4

```diff
@@ -238,7 +238,7 @@ typeSpecifier
     ;
 
 structOrUnionSpecifier
-    :   structOrUnion Identifier? '{' structDeclarationList '}'
```

Comment:
Question - why the change in the C grammar?  Is this just a new version from antlr, or did you have to rewrite the grammar yourself for some purpose?